flipperdevices/flipperzero-firmware
# [FL-1643] NFC emv assets

# What's new

- Add country and currency codes assets and parser
- Add key-value parser to File Worker
- Add frame element to widget
- Improve NFC bank card UI

# Verification 

- Upload assets/resources/nfc/emv/ folder on SD card to nfc/emv/ folder
- Verify AID, region and currency parsing

# Checklist (do not modify)
